### PR TITLE
Add delete all collections endpoint and udpate CLI (#314)

### PR DESCRIPTION
**Pull request by MarcinMaciaszek**

### CODE REVIEW

File: rfhub2/cli/keywords/keywords_importer.py [hunk 1]
```diff
@@ -48,8 +48,7 @@ def _delete_collections(self) -> Set[int]:
         """
         collections = self.client.get_collections()
         collections_id = {collection["id"] for collection in collections}
-        for id in collections_id:
-            self.client.delete_collection(id)
+        self.client.delete_all_collection()
```
Comment (MaciejWiczk) [inline rfhub2/cli/keywords/keywords_importer.py]: I think that this method might not be needed anymore at all. All it was doing is done by `delete_all_collections` method

Comment (MarcinMaciaszek) [reply rfhub2/cli/keywords/keywords_importer.py:51]: true

File: rfhub2/cli/keywords/keywords_importer.py [hunk 2]
```diff
@@ -32,26 +32,17 @@ def delete_all_collections(self) -> Set[int]:
         """
```
Comment (MaciejWiczk) [inline rfhub2/cli/keywords/keywords_importer.py]: This whole method is not needed, due to having 
```
self.client.delete_all_collections()
```

Comment (MarcinMaciaszek) [reply rfhub2/cli/keywords/keywords_importer.py]: I have removed it, it should work well still. The only thing is the coverage. Have no idea how to cover line: 110 in collections.py
@MaciejWiczk do you have any idea how to handle it? or just remove this else?

Comment (MaciejWiczk) [reply rfhub2/cli/keywords/keywords_importer.py]: Probably it's safe to go without else. About coverage, don't try to cover everything for sake of it :)

**Review by MaciejWiczk — APPROVED** [review]

**Pull request merged**